### PR TITLE
Updated addEdge

### PR DESCRIPTION
**Pull request by BrunoRosendo**

### CODE REVIEW

**Review by monkin77 — APPROVED** [review]

File: src/graph/Graph.cpp [hunk 1]
```diff
@@ -70,6 +70,7 @@ bool Graph::addEdge(const Position &sourc, const Position &dest, double weight)
     if (v1 == NULL || v2 == NULL)
         return false;
     v1->addEdge(v2, weight);
+    v2->addEdge(v1, weight);
```
Comment (monkin77) [inline src/graph/Graph.cpp:73]: We should do this on line 83 aswell (other addEdge function)

**Review by monkin77 — CHANGES_REQUESTED** [review]

**Comment by itsdomsantos** [comment]
Everything sounds ok! Gonna close this.

**Review by itsdomsantos — APPROVED** [review]
Everything's ok!

**Review by monkin77 — APPROVED** [review]
Me gusta

**Pull request merged**